src/buildercore/terraform.py

```diff
@@ -52,6 +52,29 @@
 # see https://docs.fastly.com/guides/streaming-logs/changing-log-line-formats#available-message-formats
 FASTLY_LOG_LINE_PREFIX = 'blank' # no prefix
 
+# at the moment VCL snippets are unsupported, this can be worked
+# around by using a full VCL
+# https://github.com/terraform-providers/terraform-provider-fastly/issues/7 tracks when snippets could become available in Terraform
+FASTLY_MAIN_VCL_KEY = 'main'
+FASTLY_CUSTOM_VCL = {
+    # take from https://manage.fastly.com/configure/services/4Wswrt0KnqbtpcohzrIDDx/versions/7/vcl for example
+    # not pasting it here at the moment as needs customization of values related to customer, hostnames, etc. plus maintenance
+    FASTLY_MAIN_VCL_KEY: """to be defined""",
```

Comment:
https://docs.fastly.com/guides/vcl/mixing-and-matching-fastly-vcl-with-custom-vcl#fastlys-vcl-boilerplate?

Reply:
I can paste that here if that solves the problem. Duplication seems removed through the `#FASTLY` macros